networkit/test/test_distance.py

```diff
@@ -158,6 +199,14 @@ def genERGraphs(self, n = 100, p = 0.15, seed = 42):
 					g.forEdges(lambda u, v, ew, eid: g.setWeight(u, v, random.random()))
 				yield g
 
+	"""def testJaccard(self): 
+		for g in self.genERGraphs():
+			g.indexEdges()
+			jacc = nk.distance.JaccardDistance(g,[1,2,3])
+			jaccAtt = nk.distance.JaccardSimilarityAttributizer(g,[1,2,3])
+			self.assertIsInstance(jacc.getAttribute(),list)
+			self.assertIsInstance(jaccAtt.getAttribute(),list)"""
```

Comment:
If this test is not ready yet, please mark your PR as “draft”, so that we avoid reviewing unfinished work.